src/main/com/fulcrologic/rad/database_adapters/sql/migration.clj

```diff
@@ -30,18 +30,24 @@
    :symbol   "VARCHAR(200)"
    :uuid     "UUID"})
 
+;; Add any additional data types you require here
+(def sql-type-map
+  {:text "TEXT"})
```

Comment:
I don't think we need a map. Let's make an option that allows them to just give the literal string of the data type they want. Period.

Reply:
Yeah that makes way more sense! Done